R/cpp11.R

```diff
@@ -24,6 +24,10 @@ use_cpp11 <- function() {
 
   check_cpp_register_deps()
 
+  roxygen_ns_append(
+    sprintf("@useDynLib %s, .registration = TRUE", project_name())
+  ) && roxygen_remind()
+
```

Comment:
It appears that `use_src()` does exactly this already? Can you help me understand why `use_cpp11()` needs to do that again?

https://github.com/r-lib/usethis/blob/93f4410ab93cb6e19cd6a49cd4ce33f93055b38b/R/rcpp.R#L74-L75